source/server/listener_impl.cc

```diff
@@ -415,11 +418,11 @@ void ListenerImpl::buildUdpListenerFactory(Network::Socket::Type socket_type,
     udp_listener_config_->listener_factory_ = std::make_unique<Quic::ActiveQuicListenerFactory>(
         config_.udp_listener_config().quic_options(), concurrency, quic_stat_names_);
 #if UDP_GSO_BATCH_WRITER_COMPILETIME_SUPPORT
-    // TODO(mattklein123): We should be able to use GSO without QUICHE/QUIC. Right now this causes
-    // non-QUIC integration tests to fail, which I haven't investigated yet. Additionally, from
-    // looking at the GSO code there are substantial copying inefficiency so I don't think it's
-    // wise to enable to globally for now. I will circle back and fix both of the above with
-    // a non-QUICHE GSO implementation.
+    // TODO(mattklein123): We should be able to use GSO without QUICHE/QUIC. Right now this
+    // causes non-QUIC integration tests to fail, which I haven't investigated yet.
+    // Additionally, from looking at the GSO code there are substantial copying inefficiency so
+    // I don't think it's wise to enable to globally for now. I will circle back and fix both of
+    // the above with a non-QUICHE GSO implementation.
```

Comment:
Do you know the reason behind these comment changes?  It seems like your formatter is doing something that it shouldn't be doing.

Reply:
Hmm.. I use the `.devcontainer` (VS-Code) setup Envoy ships with (without any customizations). I can undo this formatting if it's a concern, but I'm guessing something about the standard tooling has changed since this file was last edited?

Reply:
It would be good to know what caused these diffs.  Part of my reason for asking is that the multiple comment-only whitespace diffs I noticed in this PR make a very large PR even bigger and there are many of these whitespace only changes.